### PR TITLE
iOS: add HEIF texture loader

### PR DESCRIPTION
**Pull request by nekronos**
Suggestion for https://github.com/fusetools/fuselibs-public/issues/668.

This PR contains:
- [ ] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

**Comment by nekronos** [comment]
I have not integrated this into `Image`, the `ImageSource`  code is tightly coupled with `Experimental.TextureLoader`.

**Comment by nekronos** [comment]
🤔  This could be used as the textureloader on iOS. Since UIImage supports JPG, PNG and HEIF

**Comment by kusma** [comment]
I think we should finish up https://github.com/kusma/fuselibs-public/tree/bitmap instead.

**Comment by kusma** [comment]
https://github.com/kusma/fuselibs-public/commits/eeue56-feature-bitmap has the uploading-parts.

But please, no more ad-hoc features added here until we've landed the bitmap branch.

**Comment by nekronos** [comment]
Yup, I agree. Lets discuss this on monday

File: Source/Fuse.Controls.Native/iOS/HeifTextureLoader.uno [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		[Require("Source.Include", "CoreGraphics/CoreGraphics.h")]
+		[Require("Source.Include", "GLKit/GLKit.h")]
+		class UploadPromise : Promise<texture2D>
+		{
+			byte[] _bytes;
+
+			public UploadPromise(byte[] bytes) : base(UpdateManager.Dispatcher)
+			{
+				_bytes = bytes;
+				GraphicsWorker.Dispatch(Upload)
```
Comment (kusma) [inline Source/Fuse.Controls.Native/iOS/HeifTextureLoader.uno]: missing semicolon, BTW...

**Pull request closed without merging**